app/models/role.rb

```diff
@@ -1,10 +1,16 @@
 class Role < ActiveRecord::Base
   belongs_to :resource, polymorphic: true
   has_and_belongs_to_many :users, join_table: :users_roles
-
+  before_destroy :cancel
```

Comment:
We want to override destroying roles, because rolify by default deletes a role when its last user is removed.
From the app we are not able to create or delete a role, because we have fixed available roles for all conferences. And we need these roles to exist, even without users.